app/models/like.rb

```diff
@@ -1,10 +1,8 @@
 class Like < ApplicationRecord
-  belongs_to :author, class_name: 'User'
-  belongs_to :post
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
```

Comment:
**Note**
- As long as the association is directed for a model with its default name, so the class name, and foreign key attract are not necessary. just `belongs to post`is fine.

Reply:
Thanks for pointing that out 👍🏽 you're right